`make examples` returns an error

Attempting to add a new file to the `examples/` folder triggers the following error in CI:
```sh
>>> ERROR: Examples dashboard is out of sync. Fix examples or run 'make examples'
Error: Process completed with exit code 1.
```
However, `make examples` does not work, and instead produces a blank file, with the following error:
```sh
make examples
echo "<!-- DO NOT EDIT: THIS FILE IS GENERATED FROM examples/.scripts/README-text.md VIA 'make examples' -->" > examples/README.md
cat examples/.scripts/README-text.md >> examples/README.md
./examples/.scripts/run-examples.sh >> examples/README.md
./examples/.scripts/run-examples.sh: line 10: shopt: globstar: invalid shell option name
make: *** [examples/README.md] Error 1
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -115,7 +115,11 @@ jobs:
         if: matrix.operating-system != 'windows-latest'
 
   quint-examples-dashboard:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-latest, macos-latest]
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2

--- a/examples/.scripts/run-examples.sh
+++ b/examples/.scripts/run-examples.sh
@@ -5,10 +5,6 @@
 
 set -euo pipefail
 
-# Match directories with blobs recursively
-# https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html
-shopt -s globstar
-
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 EXAMPLES_DIR="${SCRIPT_DIR}/.."
 cd "$EXAMPLES_DIR"
@@ -22,4 +18,7 @@ export APALACHE_DIST=../quint/_build/apalache
 # Since the output of each run is a row in the README.md table,
 # sorting the outputs lexographically gets the results in the
 # right order.
-parallel "${SCRIPT_DIR}/run-example.sh" ::: **/*.qnt | env LC_ALL=C sort --ignore-case
+#
+# We `cut` the first 2 characters of find to remove the leading `./`, making
+# the output prettier.
+find . -name "*.qnt" | cut -c3- | parallel "${SCRIPT_DIR}/run-example.sh" | env LC_ALL=C sort --ignore-case

